src/utils.ts

```diff
@@ -87,12 +102,16 @@ class Utils {
 	}
 
 	public getPlatformConfig(platformId?: string): any {
+		logger.info("get platform config for " + platformId);
 		const platform = platformId || 'default';
 		if (platform in this.platformConfigs) {
 			return this.platformConfigs[platform];
 		}
 		const config = Utils._getPlatformConfig(platformId);
-		this.platformConfigs[platform] = config;
+		if (config.hasMultipleAccounts !== true){
```

Comment:
is this really the right place? i would move this into the transporter to select the desired array elem